Source/Core/Core/PowerPC/Jit64Common/EmuCodeBlock.cpp

```diff
@@ -54,24 +54,25 @@ void EmuCodeBlock::MemoryExceptionCheck()
   // load/store, the trampoline generator will have stashed the exception
   // handler (that we previously generated after the fastmem instruction) in
   // trampolineExceptionHandler.
-  if (g_jit->js.generatingTrampoline)
+  if (Jitx86Base::GetState().generatingTrampoline)
```

Comment:
Thoughts on storing the result of `Jitx86Base::GetState()` in a local for readability? Potentially also in other places, depending on what the concensus is here.